db/citations.yml

```diff
@@ -7366,9 +7365,11 @@ llan;RobustnessTestSpacegroupMining22:
   optannote: ''
   optwwwlanl: ''
   pages: ''
-  supplementary_info_urls: https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
-  title: Robustness Test of the spacegroupMining Space Group Determination Model for
-    atomic pair distribution functions
+  supplementary_info_urls: 
+    - https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
+  synopsis: The spacegroupMining Space Group Determination Model is shown to be robust when applied to PDFs of different choices of experimental parameters rmax, Qmax, and atomic displacement parameters.
+  tags: robustness_test, machine_learning, data_mining, space_group, pair_distribution_function
```

Comment:
thanks. great.  remove pdf (everything is pdf!).  please check for machine learning and data mining if they are in the db with abbreviations

Reply:
Done.